api/components/com_media/src/Helper/AdapterTrait.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * @package         Joomla.API
+ * @subpackage      com_media
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license         GNU General Public License version 2 or later; see LICENSE.txt
```

Comment:
All files have their copyright indented wrong 

```suggestion
 * @copyright       (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
 * @license         GNU General Public License version 2 or later; see LICENSE.txt
```